src/hotspot/os/linux/os_linux.hpp

```diff
@@ -174,6 +174,23 @@ class Linux {
   // Return the namespace pid if so, otherwise -1.
   static int get_namespace_pid(int vmid);
 
+  // Output structure for query_memory_info()
+  struct meminfo_t {
+    ssize_t vmsize;     // current virtual size
+    ssize_t vmpeak;     // peak virtual size
+    ssize_t vmrss;      // current resident set size
+    ssize_t vmhwm;      // peak resident set size
+    ssize_t vmswap;     // swapped out
+    ssize_t rssanon;    // resident set size
+    ssize_t rssfile;    // resident set size
+    ssize_t rssshmem;   // resident set size
```

Comment:
Are these comments intionally the same for all three? Seems weird.

Reply:
Nope, copy paste error. Thanks for catching.